platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/isolated/IsolatedProjectsToolingApiModelQueryIntegrationTest.groovy

```diff
@@ -152,7 +152,7 @@ class IsolatedProjectsToolingApiModelQueryIntegrationTest extends AbstractIsolat
         """
 
         when: "requesting a model together with running a task"
-        withIsolatedProjects("-Dorg.gradle.internal.isolated-projects.configure-on-demand.tasks=true")
+        withIsolatedProjects("-Dorg.gradle.internal.isolated-projects.configure-on-demand=true")
```

Comment:
💭 Why was this and the below not changed to `=tasks` like the others?

Reply:
It's because previously we had two flags: one controlling the tooling (model building) scenario and one for tasks. They could have been set independently and both of them were required for this test. We are setting the tooling flag for all IP tooling tests:
https://github.com/gradle/gradle/pull/34837/files#diff-6fa87cbce714de297b76103f9bd1eaf798cb7053a10b14f1f2426b7bbafc191dL26-R26

Now that we have a single flag, IP tests that only do tooling requests without tasks (which are most of them) can have it set to `tooling`. Whlie few tests that do both (models and tasks), require the flag to be set to `true`, like in this case.